subscription_platform/explores/daily_active_logical_subscriptions.explore.lkml

```diff
@@ -0,0 +1,27 @@
+include: "../views/daily_active_logical_subscriptions.view.lkml"
+include: "../views/logical_subscriptions.view.lkml"
+include: "../views/table_metadata.view.lkml"
+include: "/shared/views/countries.view.lkml"
+
+explore: daily_active_logical_subscriptions {
+
+  join: countries {
+    sql_on: ${daily_active_logical_subscriptions.subscription__country_code} = ${countries.code} ;;
+    type: left_outer
+    relationship: many_to_one
+  }
+
+  join: current_subscriptions {
+    from: logical_subscriptions
+    sql_on: ${daily_active_logical_subscriptions.subscription__id} = ${current_subscriptions.id} ;;
+    type: left_outer
+    relationship: many_to_one
+  }
```

Comment:
These `current_subscriptions` joins are intended to allow access to the current state of subscriptions, particularly whether they've belatedly been flagged as fraudulent.